pkg/apis/management.cattle.io/v3/authz_types.go

```diff
@@ -178,6 +179,31 @@ type GlobalRole struct {
 	// and must exactly match with one existing namespace.
 	// +optional
 	NamespacedRules map[string][]rbacv1.PolicyRule `json:"namespacedRules,omitempty"`
+
+	// Status is the most recently observed status of the GlobalRole.
+	// +optional
+	Status *GlobalRoleStatus `json:"status,omitempty"`
```

Comment:
why is Status a pointer?

Reply:
I guess for backwards compatibility, but it's just a guess.

Reply:
I thought it was required for backwards compatibility, but I tested upgrading a previous version without the status to a version with it (not as a pointer) and it worked with no issues. In fact it re-ran the reconcile logic which fills the status information. Removing the pointer